samples/AzureMQTTTwinsBMP280Sleep/AzureMQTT/Program.cs

```diff
@@ -130,7 +135,7 @@
         );
 
         Trace("Getting twin properties");
-        mqttc.Publish($"{TwinDesiredPropertiesTopic}?$rid={Guid.NewGuid()}", Encoding.UTF8.GetBytes(""), MqttQoSLevel.AtLeastOnce, false);
+        mqttc.Publish($"{TwinDesiredPropertiesTopic}?$rid={Guid.NewGuid()}", Encoding.UTF8.GetBytes(""),string.Empty, new System.Collections.ArrayList(), MqttQoSLevel.AtLeastOnce, false);
```

Comment:
why do you need to change this?
Also please do a ctrl + k +d for the formatting, there is a little space missing

Reply:
As I explain in pull request, the signature for this function has changed... please check the mqtt library version.

Reply:
you're right. I though I kept a compatible one...
You can simplify and pass null objects, no reason to create those. The code behind takes care of it: https://github.com/nanoframework/nanoFramework.m2mqtt/blob/27ee3891c54842c8a62565c26e8f67bbe7648bb9/M2Mqtt/MqttClient.cs#L2494